udata_front/theme/gouvfr/templates/macros/metadata.html

```diff
@@ -20,7 +20,8 @@
 <meta name="static-root" content="{{ static('', _burst=False) }}" />
 <meta name="api-root" content="{{ url_for('api.root') }}" />
 <meta name="api-2-root" content="{{ url_for('apiv2.root') }}" />
-<meta name="admin-root" content="{{ url_for('admin.index') }}" />
+<meta name="admin-root" content="{{ beta_admin_url_for(path='/', fallback_path='') }}" />
+<meta name="new-admin" content="{{ config.NEW_ADMIN_URL }}" />
```

Comment:
```suggestion
<meta name="new-admin-url" content="{{ config.NEW_ADMIN_URL }}" />
```

Reply:
Done in [176ac2e](https://github.com/datagouv/udata-front/pull/682/commits/176ac2e8797a4193bcfe122e55d7b609a551c17a)